### ISSUE TITLE
[BUG] only one environmen variable used in url

### ISSUE THREAD

**Issue by dwenzel** [CPS-IT/frontend-asset-handler#16]
### PHP version

PHP 7.4.26 (cli)

### Composer version

2.2.9

### Package version

0.2.3

### Operating system

linux

### Current behavior

Any `%env()%` placeholder is replaced by the first one of multiple environment variables in a configuration string.

e.g.

```shell
Asset environment: latest
Source url: https://assets.foo.bar/assets.foo.bar/latest.tar.gz
Downloading assets... Failed
```

### Expected behavior

Any environment variable should be replaced. 

```
Source url: https://assets.foo.bar/path/to/assets/latest.tar.gz
```

### Steps to reproduce

1. Set environment variables. e.g.

```
ASSET_DOMAIN=assets.foo.bar
ASSET_PATH=path/to/assets
```
2. configure source url in `assets.json
```
```json
{
  "frontend-assets": [
    {
      "source": {
        "type": "http",
        "url": "https://%env(ASSET_DOMAIN)%/%env(ASSET_PATH)%/{environment}.tar.gz"
      },
      "target": {
        "path": "app/web/assets/"
      },
      "environments": {
     ...        
       }
      }
    }
  ]
}

```
3. execute 
```
composer fetch-assets
```

**Comment by eliashaeussler** [comment]
@dwenzel Thanks for the issue. It will be resolved with #17 and included in the first stable release.

**Comment by eliashaeussler** [comment]
This fix is now part of the `1.0.0` release.
